From d85b62a5fc0c99344fe1584eaa5f0ddb5ee4f3ef Mon Sep 17 00:00:00 2001
Message-ID: <d85b62a5fc0c99344fe1584eaa5f0ddb5ee4f3ef.1694125610.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1694125609.git.sharlatanus@gmail.com>
References: <cover.1694125609.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Thu, 7 Sep 2023 21:54:39 +0100
Subject: [PATCH 17/19] gnu: python-stdatamodels: Update to 1.8.0.

* gnu/packages/astronomy.scm (python-stdatamodels): Update to 1.8.0.
[propagated-inputs]: Remove python-jsonschema.
[native-inputs]: Remove python-pytest-openfiles.
---
 gnu/packages/astronomy.scm | 6 ++----
 1 file changed, 2 insertions(+), 4 deletions(-)

diff --git a/gnu/packages/astronomy.scm b/gnu/packages/astronomy.scm
index fb501e421c..783f328c4d 100644
--- a/gnu/packages/astronomy.scm
+++ b/gnu/packages/astronomy.scm
@@ -2526,13 +2526,13 @@ (define-public python-stcal
 (define-public python-stdatamodels
   (package
     (name "python-stdatamodels")
-    (version "1.7.1")
+    (version "1.8.0")
     (source (origin
               (method url-fetch)
               (uri (pypi-uri "stdatamodels" version))
               (sha256
                (base32
-                "0d7a27myl3xgri3z7yx26mr4dpqnlfhdh5i2ql2miwymi0mx3ij5"))))
+                "003h3f07aqf6s24ivwnps81n2h51k545b236s75l8ppvjrm45xfq"))))
     (build-system pyproject-build-system)
     (arguments
      (list
@@ -2546,13 +2546,11 @@ (define-public python-stdatamodels
     (propagated-inputs (list python-asdf
                              python-asdf-astropy
                              python-astropy
-                             python-jsonschema
                              python-numpy
                              python-psutil))
     (native-inputs (list python-crds
                          python-pytest
                          python-pytest-doctestplus
-                         python-pytest-openfiles
                          python-scipy
                          python-semantic-version
                          python-setuptools-scm))
-- 
2.41.0

